.github/workflows/modules-tests.yml

```diff
@@ -12,7 +12,7 @@ jobs:
         uses: actions/checkout@v3
 
       - name: Install Avocado to run tests
-        run: pip3 install git+https://github.com/avocado-framework/avocado@7998c0a6d1bdd8ce60235cd4d2aa04afbebfe00e#egg=avocado_framework
+        run: pip3 install avocado-framework==102.0
```

Comment:
Base on the [BP005](https://avocado-framework.readthedocs.io/en/latest/blueprints/BP005.html#release) “there shouldn't be any type of release version sync between Avocado and autils”. Therefore, we shouldn't stick to a specific avocado version. What about to add only `pip3 install avocado-framework`?